best-practices.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true
+    },
+    rules: {
+        // 'accessor-pairs': 2,
+        // 'array-callback-return': 2,
+        // 'block-scoped-var': 2,
+        // NO_RULE 'class-methods-use-this': 2,
```

Comment:
I the main reason I added this is for ease of use. 

Pre this PR someone would have to cross reference the rules with the eslint site and copy and paste the config files in here (check if already exists and so on). My intention was to make it easy for someone to look at the category file (f.ex `stylistic.js`) and get a fast clear picture of what are the possible rules they can enabled.

I will add the suggested workflow to follow in a contribution guide. That would give all the information to anyone who would like to contribute.